src/views/components/admin/deleteParkingLotForm.js

```diff
@@ -0,0 +1,75 @@
+import React, { useState, useEffect } from 'react';
+import { Container, Form, Button, Alert } from 'react-bootstrap';
+import useDeleteParkingLot from '../../../controllers/useDeleteParkingLot.js';
+import useAdminGetParkingLots from '../../../controllers/useAdminGetParkingLot.js';
+
+function DeleteParkingLotForm() {
```

Comment:
Let's move these lot forms into modals and into their own pages on admin-dashboard. You can look at how parking requests are made on the driver's side for example.